tools/cli/cli.py

```diff
@@ -0,0 +1,63 @@
+# Copyright © 2025 IBM
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import io, sys
+
+from common import *
+
+class CLI:
+    def __init__(self, args):
+        self.args = args
+        if self.args['--verbose']:
+            VERBOSE = True
+
+    def command(self):
+        if self.args.get('validate') and self.args['validate']:
+            return Validate(self.args)
```

Comment:
I get `NameError: name 'Validate' is not defined` here - where is Validate defined?
We need an
```
from bee_hive.cli.validate_command import Validate
```
or similar.

Alternatively use dynamic class loading?

I think the cli also needs to be in a python package (either bee_hive, or a new one) so we can get the imports precise and robust

Tried an import, but then get Command as unknown - so presume this is work-in-progress so not yet ready?